spec.bs

```diff
@@ -1540,6 +1540,34 @@ trust that the merchant showed the user the correct amount in their checkout
 flow (and any fraud discoveries are post-payment, when the user checks their
 account statement).
 
+## Attacker generated [=Browser Bound Key=] ## {#sctn-security-attacker-generated-bbk}
+
+The [=Relying Party=] (e.g. a bank) can and should protect against attacker
+generated BBKs by [[#sctn-verifying-assertion|verifying the authentication
+assertion]] that contains the [=browser bound key|BBK=] public key and ignoring
+BBK public keys from cryptograms with invalid signatures. While there will be
+two signatures present, one from the [=SPC Credential=] and one from the BBK,
+the assertion should be verified using the passkey (in addition to the BBK).
+The BBK provides an additional signature, not a replacement for the passkey
+signature.
+
+An attacker posing as a merchant may attempt to substitute a [=browser bound
+key|BBK=] public key for another which the attacker controls the private key
+for. Later, the attacker, now impersonating the user, vists a merchant website
+and signs the SPC cryptogram using the attackers private key when the merchant
+requests SPC. The attacker has defeated the device binding aspect of the BBK.
+
+However, this attack is not feasible:
+
+1. When the attacker is subtituting the BBK: The [=Relying Party=] will detect
+    an incorrect signatures when verifying the {{CollectedClientData}} (that
+    contains the BBK public key) using the passkey public key. The relying
+    party should reject this transaction and treat this BBK public key as
+    untrusted.
+1. When the attacker attempts to impersonte the user: The [=Relying Party=]
+    will detect an incorrect signature when verifying using the passkey public
+    key. Additionally, the [=Relying Party=] does not trust this BBK.
```

Comment:
```suggestion
    key. Additionally, the [=Relying Party=] should treat this BBK as untrusted.
```

Reply:
(I'm reusing the previous formulation "should treat this BBK as untrusted." If that's not what is meant, please adjust accordingly.)